timur/lib/client/jsx/components/browser/view_items/magma_item.jsx

```diff
@@ -0,0 +1,23 @@
+// Framework libraries.
+import * as React from 'react';
+
+import AttributeViewer from '../../attributes/attribute_viewer';
+
+const isRevised = (revision, record, att_name) =>
+  att_name in revision && record[att_name] != revision[att_name];
+
+export const MagmaName = ({item, template, mode, revision, record }) => {
+  let { attribute_name, title } = item;
+  let attribute = template.attributes[attribute_name];
+
+  return <div className={ `item_name ${ (mode == 'edit' && isRevised(revision,record,attribute_name)) ? 'revised' : '' }` } title={attribute.desc}>
+    {attribute.display_name || title || attribute_name}
```

Comment:
When I navigate to a patient (and the `overview` view), I get a blank screen with this console error thrown around here:

```
No such attribute clinical in template for patient timur.bundle.js:38394:13

TypeError: attribute is undefined
    MagmaName timur.bundle.js:46781
```

Is it because the "tab" headers are being treated as attributes?

Reply:
Running into the same issue with `flow` records...I think the translation from the three different `view_` tables to `views` is not quite doing what it is expected to do?

Reply:
Hmm, the translation works, it just failed to fully account for a non-existent attribute in the template - in the default development ipi view, there is a spurious attribute "clinical" which should not exist (visible as an extra empty space at the bottom of the first pane the timur.ucsf.edu/ipi patient view). I had noted this before rebuilding the view table and removed it, thus missed this in my review.